packages/arb-util/ethutils/client.go

```diff
@@ -108,7 +113,18 @@ func (r *RPCEthClient) handleCallErr(err error) error {
 
 	// If we've had above a threshold number of errors, reinitialize the connection
 	if totalErrCount >= maxErrCount {
-		log.Warn().Err(err).Str("url", r.url).Msg("Reconnecting to client endpoint after repeated errors")
+		var partialLog *zerolog.Event
+		if totalErrCount < errorCountError {
```

Comment:
I don't think `totalErrCount` should ever reach this size because `r.reconnect` resets it. Alternatively I think we could just always make this an error. I'd guess the false positive rate would be pretty low

Reply:
removed.  Fixed actual problem instead

Reply:
It is too noisy to always have it as an error.